api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -114,10 +127,18 @@ def test_delete_github_configuration(
             github_configuration.id,
         ],
     )
+
+    mock_generate_token = mocker.patch(
+        "integrations.github.views.generate_jwt_token",
+    )
+    mock_generate_token.return_value = "mocked_token"
+    mocker.patch("requests.delete", side_effect=mocked_requests_delete)
```

Comment:
Please use the `responses` fixture to generate a response.

See example usage [here](https://github.com/Flagsmith/flagsmith/blob/e8d133726c5b724057175bfc7d945e12a733f9d1/api/tests/unit/integrations/flagsmith/test_unit_flagsmith_service.py#L98-L103).

Reply:
Done. Thanks!